js/app.js

```diff
@@ -117,16 +120,39 @@ storeForm.addEventListener('submit', addStore);
 function addStore(event) {
     event.preventDefault();
     let form = event.target;
-    let name = properCase(form.name.value);
+    let name = properCase(form.name.value); // send user submitted name to properCase() to normalize upper/lower case
```

Comment:
In JS you can actually reference elements in a string by their index the same way you access arrays. Instead of `nameString.charAt(0)` you can just type `nameString[0]`